pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -363,5 +363,7 @@
 
     @Deprecated
     public static final String TABLE_NAME = "segment.table.name";
+
+    public static final String MULTIPLE_RECORDS_KEY = "multiple.records.key";
```

Comment:
put this in GenericRow?
multiple.records.key -> $MULTIPLE_RECORDS_KEY$ avoid '.'